### ISSUE TITLE
tagline for NEST

### ISSUE THREAD

**Issue by telepath37** [insightsengineering/nest#5]
We need to create a slogan/tagline for NEST and publicize it on the website (and especially all external communications)

## Requirements

- [x] concise (< 10 words?)
- [x] Around the theme of: enabling faster and more efficient insights generation for exploratory and regulatory purposes under clinical research settings

**Comment by Melkiades** [comment]
Just trying: "Revolutionizing Clinical Research: Advancing Healthcare through Data-Driven Insights."

**Comment by yinmy0503** [comment]
Simplifying ADAMs to Analytical Reports.

**Comment by yinmy0503** [comment]
All the proposals seem attractive from a high-level perspective, one thing I am curious about: do we want to use this slogan to let the targeted end users (like ADS in ROCHE) get an intuitive idea of what NEST does and how efficient it is compared to traditional tools?  As an ADS in oncology myself, I don't quite get what NEST could help my daily work from the proposed slogan.

**Comment by Melkiades** [comment]
You are right. The current ones might be too indirect. Let me try again:
* Simplifying Clinical Reporting
* Accelerating Clinical Reporting

I think 3 words it should be to be catchy. Maybe instead of Reporting, discoveries?

**Comment by telepath37** [comment]
> All the proposals seem attractive from a high-level perspective, one thing I am curious about: do we want to use this slogan to let the targeted end users (like ADS in ROCHE) get an intuitive idea of what NEST does and how efficient it is compared to traditional tools? As an ADS in oncology myself, I don't quite get what NEST could help my daily work from the proposed slogan.

On the website homepage, we likely will provide a short blurb about NEST in general (e.g. what it offers for statistical programmers in the pharma industry, etc.), and by exploring the website, I hope it is self-explanatory enough for anyone to understand what NEST is. Personally, I think tagline is supposed to be kept at high-level, and in many cases, a bit 'abstract' (as an extreme example, 'just do it' doesn't reflect NIKE is a sportswear company, but people know it is one by browsing their website or visiting their store).

**Comment by telepath37** [comment]
> You are right. The current ones might be too indirect. Let me try again:
> 
> * Simplifying Clinical Reporting
> * Accelerating Clinical Reporting
> 
> I think 3 words it should be to be catchy. Maybe instead of Reporting, discoveries?

How about _**Democratize Clinical Reporting**_?

**Comment by lcd2yyz** [comment]
> How about _**Democratizing Clinical Reporting**_?

I don't have any ideas yet... buuuut I don't like this :confused:
Too political-sounding, and too mouthful to read. Should stick with something that can roll off the tongue easily.

**Comment by lcd2yyz** [comment]
"Let your data take flight" - clearly sticking with the nest/bird theme here :smirk:
Can I keep it for teal? :satisfied:

**Comment by Melkiades** [comment]
> "Let your data take flight" - clearly sticking with the nest/bird theme here 😏 Can I keep it for teal? 😆

I agree with you, democratizing is too political. Still, I think it is best to keep the slogan more professional as it is a general one for nest (which is not really flying :D). Surely, we can have this kind of fun mottos for single repos like `teal`. Like: "Let your data take the leap"

**Comment by telepath37** [comment]
**Accelerating Clinical Reporting** is the tagline that got most votes - we'll use it on our website as well as all future external communications.

**Comment by lcd2yyz** [comment]
This https://github.com/insightsengineering/nest/issues/13 got me thinking about the tagline again (not suggesting that we are to commit to `nest-insights.com` or anything...)

Maybe "Accelerating Clinical Insights" (as oppose to "accelerating clinical reporting") is better for a tagline? "Reporting" just sounds very stern and strict....

**Comment by telepath37** [comment]
> This #13 got me thinking about the tagline again (not suggesting that we are to commit to `nest-insights.com` or anything...)
> 
> Maybe "Accelerating Clinical Insights" (as oppose to "accelerating clinical reporting") is better for a tagline? "Reporting" just sounds very stern and strict....

Good point - I am open to change it. If we do go with nest-insights.com, I think it's good to have it aligned with the tagline.
